communication/src/main/java/org/phenotips/messaging/script/AnonymousCommunicationScriptService.java

```diff
@@ -102,4 +115,27 @@ public Connection grantAccess(String connectionId)
             return null;
         }
     }
+
+    /**
+     * When a user contacts another user regarding a match using the Similar Patients UI,
+     * mark that match as 'notified' in the match table in the administration.
+     *
+     * @param patientId reference patientID
+     * @param matchId matched patient ID
+     */
+    private void setNotified(String patientId, String matchId)
```

Comment:
Why can't this be done inside `actionManager`? Script services shouldn't contain this much logic.